src/components/app/App.js

```diff
@@ -1,37 +1,36 @@
-import 'abortcontroller-polyfill/dist/polyfill-patch-fetch';
-import 'typeface-roboto';
-import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import i18n from '@dhis2/d2-i18n';
-import { useDataEngine } from '@dhis2/app-runtime';
-import { AlertsProvider } from '@dhis2/app-service-alerts';
-import { useSetting } from '@dhis2/app-service-datastore';
-import { CssReset, CssVariables, HeaderBar } from '@dhis2/ui';
-import isEmpty from 'lodash/isEmpty';
-import AppMenu from './AppMenu';
-import { useSystemSettings } from '../SystemSettingsProvider';
-import LayersPanel from '../layers/LayersPanel';
-import LayersToggle from '../layers/LayersToggle';
-import MapContainer from '../map/MapContainer';
-import BottomPanel from '../datatable/BottomPanel';
-import LayerEdit from '../edit/LayerEdit';
-import ContextMenu from '../map/ContextMenu';
-import OrgUnitProfile from '../orgunits/OrgUnitProfile';
-import AlertStack from '../alerts/AlertStack';
-import InterpretationsPanel from '../interpretations/InterpretationsPanel';
-import DataDownloadDialog from '../layers/download/DataDownloadDialog';
-import OpenAsMapDialog from '../openAs/OpenAsMapDialog';
-import FatalErrorBoundary from '../errors/FatalErrorBoundary';
-import { tSetAnalyticalObject } from '../../actions/analyticalObject';
-import { tSetOrgUnitTree } from '../../actions/orgUnits';
-import { tOpenMap } from '../../actions/map';
-import { tSetExternalLayers } from '../../actions/externalLayers';
-import { removeBingBasemaps, setBingMapsApiKey } from '../../actions/basemap';
-import { CURRENT_AO_KEY } from '../../util/analyticalObject';
-import { getUrlParameter } from '../../util/requests';
-
-import styles from './styles/App.module.css';
+import 'abortcontroller-polyfill/dist/polyfill-patch-fetch'
+import 'typeface-roboto'
+import { useDataEngine } from '@dhis2/app-runtime'
+import { AlertsProvider } from '@dhis2/app-service-alerts'
+import { useSetting } from '@dhis2/app-service-datastore'
+import i18n from '@dhis2/d2-i18n'
+import { CssReset, CssVariables, HeaderBar } from '@dhis2/ui'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import { tSetAnalyticalObject } from '../../actions/analyticalObject.js'
+import { removeBingBasemaps, setBingMapsApiKey } from '../../actions/basemap.js'
+import { tSetExternalLayers } from '../../actions/externalLayers.js'
+import { tOpenMap } from '../../actions/map.js'
+import { tSetOrgUnitTree } from '../../actions/orgUnits.js'
+import { CURRENT_AO_KEY } from '../../util/analyticalObject.js'
+import { getUrlParameter } from '../../util/requests.js'
+import AlertStack from '../alerts/AlertStack.js'
```

Comment:
Possible to avoid lining warnings for this file? 

![Screenshot 2022-12-12 at 17 17 55](https://user-images.githubusercontent.com/548708/207102281-389bd1b8-7198-4b90-b4d3-0e6e496667ca.png)

Reply:
Det kommer i en ny PR 🎉